### PR TITLE
fix cuak grass icons

### PR DESCRIPTION
**Pull request by Chubbygummibear**
he asked me to do this, tested and it works now

# Changelog

Edit the changelog below to reflect the changes made by this PR, even if the changes are minor - required for every PR.
If you add a name after the ':cl', that name will be used in the changelog. Leave it empty to use your GitHub name.

:cl:  Cuackles, Chubbygummibear
bugfix: grass icons for holodeck fixed
/:cl:

### CODE REVIEW

**Review by ToasterBiome — CHANGES_REQUESTED** [review]
no amke them do the random grass

File: code/game/turfs/open.dm [hunk 1]
```diff
@@ -172,7 +172,7 @@
 /turf/open/indestructible/grass
 	name = "grass patch"
 	desc = "Yep, it's grass."
-	icon_state = "grass"
+	icon_state = "grass1"
```
Comment (ToasterBiome) [inline code/game/turfs/open.dm:175]: why is this changed

**Review by ToasterBiome — CHANGES_REQUESTED** [review]

File: code/game/turfs/open.dm (hunk 1 above)
Comment (Chubbygummibear) [reply code/game/turfs/open.dm:175]: because "grass" doesn't exist anymore. it's grass1-4

Comment (ToasterBiome) [reply code/game/turfs/open.dm:175]: > because "grass" doesn't exist anymore. it's grass1-4

sorry this needs to be in it's own separate PR

**Review by ToasterBiome — APPROVED** [review]
